src/Orleans.Core/Runtime/Constants.cs

```diff
@@ -71,38 +66,36 @@ internal class Constants
 
         public static readonly TimeSpan DEFAULT_CLIENT_DROP_TIMEOUT = TimeSpan.FromMinutes(1);
 
-        private static readonly Dictionary<GrainId, string> singletonSystemTargetNames = new Dictionary<GrainId, string>
-        {
-            {DirectoryServiceId, "DirectoryService"},
-            {DirectoryCacheValidatorId, "DirectoryCacheValidator"},
-            {SiloControlId,"SiloControl"},
-            {ClientObserverRegistrarId,"ClientObserverRegistrar"},
-            {CatalogId,"Catalog"},
-            {MembershipOracleId,"MembershipOracle"},
-            {MultiClusterOracleId,"MultiClusterOracle"},
-            {TypeManagerId,"TypeManagerId"},
-            {ProtocolGatewayId,"ProtocolGateway"},
-            {FallbackSystemTargetId, "FallbackSystemTarget"},
-            {DeploymentLoadPublisherSystemTargetId, "DeploymentLoadPublisherSystemTarget"},
-        };
-
-        private static readonly Dictionary<int, string> nonSingletonSystemTargetNames = new Dictionary<int, string>
+        private static readonly Dictionary<GrainType, string> singletonSystemTargetNames = new Dictionary<GrainType, string>
```

Comment:
Not really important: maybe we don't need this anymore with the new `GrainType` ?

Reply:
Should we remove it later?